lib/core/src/Cardano/Wallet/Api/Link.hs

```diff
@@ -237,6 +238,21 @@ getUTxOsStatistics w = discriminate @style
   where
     wid = w ^. typed @(ApiT WalletId)
 
+getWalletUtxoSnapshot
+    :: forall (style :: WalletStyle) w.
+        ( HasCallStack
+        , Discriminate style
+        , HasType (ApiT WalletId) w
+        )
+    => w
+    -> (Method, Text)
+getWalletUtxoSnapshot w = discriminate @style
+    (endpoint @Api.GetWalletUtxoSnapshot (wid &))
+    (endpoint @Api.GetByronWalletUtxoSnapshot (wid &))
+    (notSupported "Shared")
```

Comment:
Why not supported for shared wallets? They are have UTxO, once active.

Reply:
> Why not supported for shared wallets? They are have UTxO, once active.

Good point.

@piotr-iohk, do you know of a good reason to not support shared wallets?

Reply:
Nope, my guess was that we will support them eventually at some point (once they are active). :)
If it is possible to wire them up right now then, yes, why not.